CMakeLists.txt

```diff
@@ -318,7 +318,16 @@ else()
     usFunctionCheckCompilerFlags(-Wno-mismatched-tags US_CXX_FLAGS)
   endif()
 
-  usFunctionCheckCompilerFlags("-fvisibility=hidden -fvisibility-inlines-hidden" _have_visibility)
+  # Issue #56: Workaround a clang compiler issue (https://llvm.org/bugs/show_bug.cgi?id=26749).
+  # The typeinfo symbols do not have reliable symbol visibility. Instead of exporting all
```

Comment:
Maybe I misunderstand things here, but this reads like we were exporting all symbols before. Shouldn't this rather read like: Instead of exporting only symbols (and their associated typeinfo symbols) which have an explicit "default" visibility, export all typeinfo symbols by default but keep all other  symbols hidden (unless explicitly overridden).